openmls/src/key_packages/mod.rs

```diff
@@ -389,25 +374,14 @@ impl KeyPackage {
             init_key: hpke_init_key,
             leaf_node,
             credential: credential_bundle.credential().clone(),
-            extensions,
+            extensions: vec![],
```

Comment:
Here, I think, we don't want to include the extensions because there are no key package extensions defined.